tests/access_tests.py

```diff
@@ -348,6 +348,50 @@ def test_request_access(self):
         gamma_user.roles.remove(sm.find_role('dummy_role'))
         session.commit()
 
+    def test_update_role(self):
```

Comment:
is it typical in python to have one test that asserts multiple things? would it make sense to break this into multiple tests?